### PR TITLE
Use hopefully soon-to-be-official cljfmt lib entrypoint

### PR DESCRIPTION
**Pull request by cap10morgan**

### CODE REVIEW

**Comment by cap10morgan** [comment]
* **#58** <a href="https://app.graphite.dev/github/pr/fluree/server/58?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#57** <a href="https://app.graphite.dev/github/pr/fluree/server/57?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @cap10morgan and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: bb/lib/cljfmt.clj [hunk 1]
```diff
@@ -11,8 +11,9 @@
   "Runs cljfmt check on all files (recursively) in dir. Returns a collection of
   any files that failed the check."
   [dir]
-  (let [cljfmt-opts {:paths [dir], :diff? false}
-        {:keys [counts incorrect error] :as _result} (fmt/check cljfmt-opts)]
+  (let [cljfmt-opts {:diff? false}
```
Comment (zonotope) [inline bb/lib/cljfmt.clj:14]: this isn't involved in this pr, but is this option to display a diff in the output? if that's the case, that seems helpful.

**Review by zonotope — APPROVED** [review]

File: bb/lib/cljfmt.clj (hunk 1 above)
Comment (cap10morgan) [reply bb/lib/cljfmt.clj]: It is, but I think in practice it's much more noise than signal. It still outputs messages saying where the problem is and a description of the specific problem, and that's always been more than sufficient to know what I need to do to fix it.

Here's an example:
```
clj-kondo lint:
src/fluree/server/consensus.clj:12:19-25::warning - unused binding thingy
```

**Comment by cap10morgan** [comment]
@zonotope OK this is now using the upstream merged code. Hopefully it stays stable for a while, but [who knows](https://github.com/weavejester/cljfmt/pull/342#issuecomment-2154894525). Look OK?

**Review by zonotope — APPROVED** [review]
🎄

**Comment by cap10morgan** [comment]
### Merge activity

* **Jun 21, 2:57 PM EDT**: @cap10morgan started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/fluree/server/57).

**Pull request merged**